src/screens/PasswordRecovery/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import EmailForm from './componentes/EmailForm'
+import { isValidEmail } from '../authentication/utils'
+import * as firebase from 'firebase'
+import { View, KeyboardAvoidingView } from 'react-native'
+import { SvgXml } from 'react-native-svg'
+import { Title, Description } from '~/components/texts'
+import styles from './styles'
+import Image from './resources/recover.svg'
```

Comment:
Please move svgs to the `~/assets` folder. Also when importing it, consider using a `camelCase` notation, `Image` sounds like a component or class name.